bin/build-docker.sh

```diff
@@ -3,7 +3,14 @@
 VERSION=`jq -r ".version" package.json`
 SERIES=${VERSION:0:4}-latest
 
-cat package.json | grep -v electron > server-package.json
+if ! [[ $(which tsc) ]]; then
+    npm i -g typescript
+fi
```

Comment:
```suggestion
```

Not needed, since we can use `npx`.

Reply:
This was added to make sure that tsc could run and convert all the `.ts` to `.js` which removed all the "CommonJS" vs  "ESNext" errors I was getting. After making the rest of the changes, I'll come back to this and do some testing with `npx`.

Reply:
Removed.